Update pages.md (#237)
directus/docs: extensions/pages.md

A Page is made up of at least two core files. You can create a page from scratch or use the [extension toolkit](https://github.com/directus/extension-toolkit) to generate boilerplate code.

If not using the extension toolkit, here's how you can create a page from scratch:

> 1. Create a meta.json file in `public/extensions/custom/pages/<your custom page folder`. See below for an example.

Reviewer: ```suggestion
1. Create a meta.json file in `public/extensions/custom/pages/<your custom page folder>`. See below for an example.
```

> 2. Write up your page in a .vue file, this must be named "page.vue".

Reviewer: ```suggestion
2. Write up your page in a `page.vue`. The app will explicitly look for the resulting built `page.js` file, so make sure to use `page.vue` as the name for this file.
```

> 3. Transpile the .vue file in the same directory as step #1.

Reviewer: ```suggestion
3. Transpile the `page.vue` file in the same directory as step #1.
```
